cedar-policy/src/ffi/validate.rs

```diff
@@ -478,26 +515,39 @@ mod test {
                 }
               }
             }
-          }}},
-          "policies": "forbid(principal, action, resource);permit(principal == Photo::\"photo.jpg\", action == Action::\"viewPhoto\", resource == User::\"alice\");"
+          }},
+          "policies": {
+            "staticPolicies": "forbid(principal, action, resource);permit(principal == Photo::\"photo.jpg\", action == Action::\"viewPhoto\", resource == User::\"alice\");"
+          }
         });
 
-        assert_validates_with_errors(json, 1);
+        let errs = assert_validates_with_errors(json);
+        assert_length_matches(&errs, 1);
+        assert_eq!(errs[0].policy_id, PolicyId::new("policy1"));
```

Comment:
I think you made a helper for this

Reply:
Helper is a little different since it takes a `DetailedError` instead of `ValidationError`